reference/glossary.rst

```diff
@@ -659,19 +659,33 @@ Glossary
         Related topic(s): Backups and version control, and Security.
 
     HA
-        *Work in Progress*
+        High Availability (HA) is the process of ensuring that a system is always up. To achieve this, a redundant system is setup that either
+        takes over when the main system is down or run alongside the main system to load balancing the workload.
+
+        Related topic(s): High availability, and OpenLDAP.
 
     HBAs
-        *Work in Progress*
+        Host Bus Adapters (HBAs) are interface cards that connects a server to a storage device.
+
+        Related topic(S): Device mapper multipathing and Server installation
 
     HMAC
-        *Work in Progress*
+        A Hash-based Message Authentication Code (HMAC) is a type of Message Authentication Code (MAC).
+        A MAC verifies that a message hasn't been modified during transmission by combining a shared secret key between the sender and reciever, and a hash function.
+        While a general MAC may use various techniques during combination, HMAC follows a structured way. When a message and its 
+        HMAC are sent, the receiver verifies the integrity by computing the HMAC again - if the message is altered, the value will differ.
+
+        Related topic(S): High availability, Kerberos, Cryptography, and Security.
 
     HMC
-        *Work in Progress*
+        Hardware Management Console (HMC) is used to manage IBM servers. It can handle a task like configuring network settings, loading Ubuntu installation files and installing the OS.
```

Comment:
```suggestion
        Hardware Management Console (HMC) is used to manage IBM servers. It can handle tasks like configuring network settings, loading Ubuntu installation files and installing the OS.
```